lib/standards/html-elms.js

```diff
@@ -674,7 +674,7 @@ const htmlElms = {
     noAriaAttrs: true
   },
   picture: {
-    contentTypes: ['embedded', 'phrasing', 'flow'],
+    contentTypes: ['phrasing', 'flow'],
```

Comment:
This was a mistake. `figure` is not am embedded element. The `img` inside it is. You can put text inside a picture along side the img and the text will be rendered. It also isn't described as embedded in the HTML spec:
https://html.spec.whatwg.org/multipage/grouping-content.html#the-figure-element